packages/pigeon/pigeons/core_tests.dart

```diff
@@ -36,6 +59,49 @@ class AllTypesWrapper {
   AllTypes values;
 }
 
+// A class containing all supported nullable types.
+class AllNullableTypes {
+  AllNullableTypes(
+    this.aNullableBool,
+    this.aNullableInt,
+    this.aNullableDouble,
+    this.aNullableString,
+    this.aNullableByteArray,
+    this.aNullable4ByteArray,
+    this.aNullable8ByteArray,
+    this.aNullableFloatArray,
+    this.aNullableList,
+    this.aNullableMap,
+    this.nullableNestedList,
+    this.nullableMapWithAnnotations,
+    this.nullableMapWithObject,
+    this.aNullableEnum,
+  );
+
+  bool? aNullableBool;
+  int? aNullableInt;
+  double? aNullableDouble;
+  String? aNullableString;
+  Uint8List? aNullableByteArray;
+  Int32List? aNullable4ByteArray;
+  Int64List? aNullable8ByteArray;
+  Float64List? aNullableFloatArray;
+  // ignore: always_specify_types, strict_raw_type
+  List? aNullableList;
+  // ignore: always_specify_types, strict_raw_type
+  Map? aNullableMap;
+  List<List<bool?>?>? nullableNestedList;
+  Map<String?, String?>? nullableMapWithAnnotations;
+  Map<String?, Object?>? nullableMapWithObject;
+  AnEnum? aNullableEnum;
+}
+
+// A class for testing nested object handling.
+class AllNullableTypesWrapper {
+  AllNullableTypesWrapper(this.values);
+  AllNullableTypes values;
```

Comment:
If we want to test something different here, this should be nullable. Otherwise, I would just remove this whole class (since I'm not sure there's anything especially distinct about nullable vs non-nullable nested objects).

Reply:
I removed the non nullable version of this. Again, to make instantiating classes cleaner as the implementation you created only uses a string, it seemed wasteful to create a full non null class